manifests/configfile.pp

```diff
@@ -40,6 +43,7 @@
   Optional[String] $content = undef,
   Stdlib::Absolutepath $config_dir = $caddy::config_dir,
   Optional[Stdlib::Absolutepath] $enable_dir = $caddy::config_enable_dir,
+  String $file_extension = $caddy::config_file_extension,
```

Comment:
do we want to allow empty string here?

Reply:
yes, I did it intentionally.. if someone would like to include any file in a config directory irregardless of extension